### PR TITLE
BETA: Register horn7x.is-a.dev

### PR DESCRIPTION
**Pull request by med14508**
Added `horn7x.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/hornbyte.json
Comment (VaibhavSys) [inline domains/hornbyte.json:1]: Empty file

File: hornbyte.json
Comment (VaibhavSys) [inline hornbyte.json:1]: Empty file and outside domains directory

**Review by VaibhavSys — DISMISSED** [review]
By the way, this is weird
image

File: hornbyte.json
Comment (horn7x) [reply hornbyte.json:1]: hey yeah ik but i no longer need that domain i forgot to remove it

Comment (horn7x) [reply hornbyte.json:1]: about the website, just forget it im updating it, that was like a website i made in 20minutes, im currently making a better one to be the Official one

**Review by wdhdev — APPROVED** [review]

**Pull request merged**